### PR TITLE
Update sidecar injection doc

### PR DESCRIPTION
**Pull request by hanxiaop**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/common-problems/injection/index.md [hunk 1]
```diff
@@ -106,22 +106,21 @@ of injected sidecar when it was.
...
+    The deployment’s metadata is ignored. Label value
     of `true` forces the sidecar to be injected while a value of
     `false` forces the sidecar to _not_ be injected.
 
-    The following annotation overrides whatever the default `policy` was
+    The following label overrides whatever the default `policy` was
     to force the sidecar to be injected:
 
     {{< text bash yaml >}}
     $ kubectl get deployment sleep -o yaml | grep "sidecar.istio.io/inject:" -C3
```
Comment (frankbu) [inline content/en/docs/ops/common-problems/injection/index.md:118]: I don't think the output, below, is what you would get with `grep -C3`. Shouldn't it be 3 lines before and after the matched line?

Comment (ericvn) [reply content/en/docs/ops/common-problems/injection/index.md:118]: I agree with Frank. The output for the command needs to be redone.

Comment (hanxiaop) [reply content/en/docs/ops/common-problems/injection/index.md]: Thanks for reminding. Revised.

**Review by ericvn — APPROVED** [review]
Thanks

**Pull request merged**